docs/source/kedro_project_setup/configuration.md

```diff
@@ -218,6 +231,43 @@ The output Python dictionary will look as follows:
 Although Jinja2 is a very powerful and extremely flexible template engine, which comes with a wide range of features, we do not recommend using it to template your configuration unless absolutely necessary. The flexibility of dynamic configuration comes at a cost of significantly reduced readability and much higher maintenance overhead. We believe that, for the majority of analytics projects, dynamically compiled configuration does more harm than good.
 ```
 
+## Configuration with OmegaConf
+
+[OmegaConf](https://omegaconf.readthedocs.io/) is a python library for dealing with configuration. It is a YAML based hierarchical configuration system, with support for merging configurations from multiple sources.
+From Kedro 0.18.5 you can use the [`OmegaConfLoader`](/kedro.config.OmegaConfLoader) which uses `OmegaConf` under the hood to load data.
+
+The `OmegaConfLoader` can load `YAML` and `JSON` files. Acceptable file extensions are `.yml`, `.yaml`, and `.json`. By default, any configuration files used by the config loaders in Kedro are `.yml` files.
+
+To use the `OmegaConfLoader` in your project, set the `CONFIG_LOADER_CLASS` constant in your [`src/<package_name>/settings.py`](settings.md):
+
+```python
+from kedro.config import OmegaConfLoader  # new import
+
+CONFIG_LOADER_CLASS = OmegaConfLoader
+```
+
+### Templating for parameters
+Templating or [variable interpolation](https://omegaconf.readthedocs.io/en/2.3_branch/usage.html#variable-interpolation), as it's called in `OmegaConf`, for parameters works out of the box as long as the name of the file containing the template values follows the same config pattern specified for parameters.
+By default, the config pattern for parameters is: `["parameters*", "parameters*/**", "**/parameters*"]`. 
+Suppose you have one parameters file called `parameters.yml` containing parameters with `omegaconf` placeholders like this:
+ 
+```yaml
+model_options:
+  test_size: ${data.size}
+  random_state: 3
+```
+
+and a file containing the template values called `parameters_globals.yml`:
+```yaml
+data:
+  size: 0.2
+```
+
+Since both of the file names (`parameters.yml` and `parameters_globals.yml`) match the config pattern for parameters, the `OmegaConfLoader` will load the files and resolve the placeholders correctly. 
+
+```{note}
```

Comment:
Aha! I see the note now. I think this should move to the top and also be more specific about the version you can find it without using a development version.

Reply:
Good point! These docs will be released together with the `OmegaConfLoader`, but in case anyone is reading the docs from `main` it's good to be clear about what version this functionality is in.